src/components/include/protocol_handler/session_observer.h

```diff
@@ -54,6 +54,58 @@ namespace protocol_handler {
  */
 enum { HASH_ID_NOT_SUPPORTED = 0, HASH_ID_WRONG = 0xFFFF0000 };
 
+/**
+ * @brief Struct with data containing attributes of starting session
+ **/
+struct StartingSessionContext {
+  transport_manager::ConnectionUID connection_id_;
+  uint8_t initial_session_id_;
```

Comment:
why initial_session_id_ is unit8, new_session_id_ is uint32?

Reply:
@yang1070 seems this difference was in old code before this pull request. Both values contains some session id value, so uint8_t is enough for both of them. Updated in https://github.com/smartdevicelink/sdl_core/commit/36306024c326aad930a636931f64f00928fce51e